packages/cli/README.md

```diff
@@ -0,0 +1,151 @@
+blitz-cli
```

Comment:
Please add a `developer` story to the README.md; it's not apparent how I am to build/test/release, etc. As a new contributor, I should be able to find this info with little friction.